app/Http/Controllers/BeatmapDiscussionsController.php

```diff
@@ -92,13 +93,16 @@ public function index()
         $search = BeatmapDiscussion::search($params);
 
         $query = $search['query']->with([
-            'user',
+            'beatmap',
+            'beatmapDiscussionVotes',
             'beatmapset',
             'startingPost',
         ])->limit($search['params']['limit'] + 1);
 
-        $discussions = new Paginator(
-            $query->get(),
+        $discussions = $query->get();
```

Comment:
This contains one extra item which is used to tell `Paginator` whether or not there are more pages. The last item shouldn't be actually rendered.